packages/mathtype-html-integration-devkit/test/unit-telemetry-checkUUIDformat.test.js

```diff
@@ -1,7 +1,14 @@
 import Telemetry from '../src/telemetry';
 
-describe('Check the correct format of the Telemetry compose UUID function',
+/**
+ * This test is docused on checking one isolated Telemetry function
+ */
+describe('Check the correct format of the Telemetry compose UUID function. TAG = Telemetry',
```

Comment:
Is the "TAG = ..." tagging format documented or taken from somewhere?

Reply:
Yes, in the refator testing for html-integrations document  it's mentioned

Reply:
I was refering more to external sources or Jest documentation. But if seems fine to me.

Reply:
The external documentation just explains that exists a command to run all the tests with the same name in the describe tag, for example run all the tests that have 'telemetry' on their description. Because of that i decided to place the TAG on the description